src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -290,14 +290,18 @@ export default {
 		object-fit: cover;
 	}
 
+	.loading {
+		display: inline-block;
+		height: 128px;
```

Comment:
I think we once also discussed to use more height, e.g. 300? cc @jancborchardt 
But not sure what the endpoint does if the image is actually smalled?

Reply:
I can check with it. otherwise there's a "fill=true" option which I'm not sure what it does, might fit

Reply:
I've tested with a picture of 50x50 and the preview provider doesn't make it bigger.
However the image tag will scale it up, so it will have the same height as the other pictures:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/277525/97202248-a92fd080-17b3-11eb-8116-974d08d233a9.png">

Reply:
okay I found the ticket with the decision to make it 384px: https://github.com/nextcloud/spreed/issues/3746#issuecomment-641979030

I'll adjust

Reply:
with new height:
<img width="733" alt="image" src="https://user-images.githubusercontent.com/277525/97223039-84495680-17cf-11eb-9837-90c7310965d7.png">

and for a wide image:
<img width="733" alt="image" src="https://user-images.githubusercontent.com/277525/97223288-de4a1c00-17cf-11eb-8a23-381566b5a479.png">